packages/asc-ui/src/components/Header/HeaderLogoTextStyle.ts

```diff
@@ -1,10 +1,6 @@
 import styled from 'styled-components'
 
-type Props = {
```

Comment:
We're no longer offering the `tall`  version of the header? Why is that?

Reply:
because the prop isn't used in this component :)